### ISSUE TITLE
Flow type value missing semi-colon terminator

### ISSUE THREAD

**Issue by gf3** [pangloss/vim-javascript#517]
Example snippet:

``` js
type State = {
  errorMessages: Array<string>;
  forceValidation: bool;
  invalidFields: Array<string>;
  submitting: bool;
};


//-----------------------------------------------------------------------------
// Blah blah
//-----------------------------------------------------------------------------

function lol() {
   // ...
}
```

can fix by changing:

``` vim
syntax region  jsFlowTypeValue      contained matchgroup=jsFlowNoise start=/:/       end=/[,}]/ contains=@jsFlowCluster
```

to

``` vim
syntax region  jsFlowTypeValue      contained matchgroup=jsFlowNoise start=/:/       end=/[,};]/ contains=@jsFlowCluster
```

**Comment by amadeus** [comment]
#518 should fix this problem. But that branch also includes some bigger fixes as well, so it would be helpful for you to test that branch and see if I didn't cause any other inadvertent issues.

**Comment by gf3** [comment]
tested and fixed in #518

**Comment by amadeus** [comment]
Fixed in `develop`
